features/networking/egress-ip.feature

```diff
@@ -2,6 +2,7 @@ Feature: Egress IP related features
 
   # @author bmeng@redhat.com
   # @case_id OCP-15465
+  @admin
```

Comment:
Do we need `@admin` here? How did it work in the past?

Reply:
Interesting, seems the first step `Given I select a random node's host` need `@admin`, but I cannot tell how did it work in the past :(
```
Given /^I select a random node's host$/ do
  ensure_admin_tagged
```

Reply:
I found the initial PR: https://github.com/openshift/cucushift/pull/6792/files, maybe it is helpful but seems no logs after the review/update.

Reply:
I just rerun the tests against V3 branch, it entered the same issue as master branch.  Here is the log errors:
Given I select a random node's host                                   # features/step_definitions/node.rb:9
      tag scenario @admin as you use admin access (RuntimeError)
      /home/jenkins/workspace/Runner-v3/lib/world.rb:66:in `ensure_admin_tagged'

It seems "Given I select a random node's host " really need @admin tag.
I am not sure how the original codes pass the tests,  but now after adding @admin tag, the case can pass.